sys/dev/hyperv/netvsc/if_hn.c

```diff
@@ -861,7 +861,7 @@ hn_set_hlen(struct mbuf *m_head)
 
 		PULLUP_HDR(m_head, ehlen + sizeof(*ip6));
 		ip6 = mtodo(m_head, ehlen);
-		if (ip6->ip6_nxt != IPPROTO_TCP) {
+		if ((ip6->ip6_nxt != IPPROTO_TCP) & (ip6->ip6_nxt != IPPROTO_UDP)) {
```

Comment:
Did you mean to do a bitwise AND here? Should this the below pattern instead (I removed the superfluous parentheses)?

```if (ip6->ip6_nxt != IPPROTO_TCP && ip6->ip6_nxt != IPPROTO_UDP) {```